addons/contour/1.6.0/Manifest

```diff
@@ -0,0 +1,3 @@
+image contour docker.io/projectcontour/contour:v1.6.0
+image contour docker.io/projectcontour/contour:latest
```

Comment:
why latest? nothing should use latest

Reply:
@laverya Its because this is being used in a `Job`, see this comment.

https://github.com/replicatedhq/kURL/pull/584/files#diff-9d2f620150ba2477c9255aabc7acadd0R994-R997